content/posts/newsletter/Newsletter for 2020-01-10.md

```diff
@@ -0,0 +1,167 @@
+
+Fabian Sauter known as COM8 (and developer of [UWPX](https://uwpx.org/)) has finalised his Bachelor Thesis on "[Smart Building Control with XMPP for IoT](https://home.in.tum.de/~sauterf/html-data/Thesis_Smart_Building_Control_with_XMPP_for_IoT.pdf)". Congratulations! The development will be continued with the goal of an official XEP.
+
+Anoxinon e.V who intend to inform and improve the situation about privacy and free software as well run a XMPP Server are looking for [new board members](https://anoxinon.de/blog/logbuchspezial_zukunftdesvereins/).
+
+Paul Schaub has written two pieces:
+* [Pitfalls for OMEMO Implementations – Part 1: Inactive Devices](https://blog.jabberhead.tk/2019/12/13/pitfalls-for-omemo-implementations-part-1-inactive-devices/)
+* [Re: The Ecosystem is Moving](https://blog.jabberhead.tk/2019/12/29/re-the-ecosystem-is-moving/) as an answer to Moxie Marlinspike's talk at 36C3 (Moxie is the creator of Signal)
+
+![The Ecosystem is Moving](https://framapic.org/vnIr36vc8UVR/V2BVA5KG8ozl.png)
+
+Alex Gnauck wrote about  [Simpler code in handlers using pattern matching](https://www.ag-software.net/2019/12/30/simpler-code-in-handlers-using-pattern-matching/) in MatriX.
+
+During the 36C3 a bunch of people finalized the idea to form a [SIG dedicated to end-to-end encryption](https://mail.jabber.org/pipermail/standards/2019-December/036769.html).
+
+### Newsletter translations 
+
+* [French](https://linuxfr.org/news/lettre-d-information-xmpp-3-decembre-2019-xmpp-dans-toutes-les-langues)
+* [German](https://www.jabber.de/xmpp-newsletter-xmpp-auf-allen-sprachen-03-dez-2019/)
+* [Spanish](https://www.jabber.de/es-xmpp-newsletter-xmpp-en-todos-los-idiomas-03-dic-2019/)
+
+## Software releases 
+
+### Servers 
+
+The Ignite Realtime community has released:
+
+* [Openfire 4.5.0 Beta](https://discourse.igniterealtime.org/t/openfire-4-5-0-beta-release/86888)
+* [Fastpath Openfire plugin 4.5.0 Beta](https://discourse.igniterealtime.org/t/fastpath-openfire-plugin-4-5-0-beta-release/86895)
+
+### Clients and applications
+
+Anu Pokharel the developer of Monal has made significant changes to its [MUC implemenation](https://monal.im/blog/group-chat-improvements/) which improves the situation for iOS ([4.1 and 4.2](https://monal.im/blog/ios-4-1-and-4-2/) and [4.2 final beta](https://monal.im/blog/ios-4-2-final-beta/)) and macOS ([2.5 beta 1](https://monal.im/blog/mac-2-5-beta-1/) [2.5 beta 2](https://monal.im/blog/mac-2-5-beta-2/)) users. Furthermore there has been many [bug fixes](https://monal.im/blog/cve-fixes/), including [HTTP upload](https://monal.im/blog/http-upload/), and [the end of iOS9 support](https://monal.im/blog/the-end-of-ios-9-support/).
```

Comment:
`implementation` typo

Reply:
Also, you don't include the OMEMO posts ( https://monal.im/blog/omemo-fixes-catalyst-and-france/ & https://monal.im/blog/omemo-is-broken-in-general-across-the-ecosystem/ ) because they're not in December? 
I find them pretty important though since they have implications in the whole ecosystem and this needs faster coordination between multiple projects, hence the news needs to be put out there.

Reply:
These are from January... We are late, indeed.